Topics/Software_Engineering/Retro.md

```diff
@@ -0,0 +1,88 @@
+
+Try and set up a retro board to allow ideas to be organised and represented in a modular structure. It can be physical or digital, but it should be in place during the meeting so that team members can have a visual aid to look towards when summarizing the good and bad that happened during the sprint. Feel free to use one of the [templates](#atlassian---fun-retro-templates) as an idea on how to set up your board.
+
+As a bonus, try to have an ice breaker set up so that people can start off with a lighthearted discussion. The best way to induce people to speak their minds is to bring up a topic everyone is interested in. Maybe have a silly question that everyone can answer, like have them compare the sprint to a random object, person, or time period.
+
+### During the Retro Meeting
+
+During the retro meeting, try to get your team members talking. It doesn't have to be at the very start, but try to get them warmed up to speak. A retro is a team effort, and requires all members to be willing to have some degree of input on the topic. Use an ice-breaker, or try and have the team do a light activity like a single round of pictionary. Try to play to your team's strengths and have them enjoy the process.
+
+Once the team is warmed up, start to introduce the data points you collected. Present to them the number of tasks completed, and ask them for a progress report for the goals your team created for themselves during the sprint.
+
+It is important during the sprint to provide visual aids and concrete numbers to help members get an overall feel for what happened during the sprint. Get people to summarize what they see, and try to get them to summarize the overall tasks they feel were done. Though it may feel unhelpful, it forces your team to think about their progress and tests them on whether they are properly staying up to date with the project and aligned with the goals they promised themselves.
+
+Once tasks and project goals were discussed, open up the meeting for discussions. This should take up the bulk of your meeting. Ask the team what went wrong during sprint? What went right? If you have comments and concerns that were brought up before the meeting, introduce them to the team and ask them if there is a current solution to said issue. Always look to get context about any issues, however, as making assumptions can easily lead to miscommunication and misunderstandings.
+
+If you have any issues or concerns brought up from previous retros, now is the time to ask the team on whether any potential solutions were thought up of that could help alleviate those issues.
+
+Don't forget to celebrate any successes that occur, no matter how small. It's important for the team to understand what was done that was successful, so that those actions can become reproduceable!
+
```

Comment:
It could be helpful to have more guidance on how to handle delicate subjects or interpersonal conflicts that may come up during retrospectives. This would make it easier for facilitators to keep the environment upbeat and productive even when addressing potentially divisive topics.

Reply:
Hmm, this is a tricky thing to navigate, as things are usually case-by-case. Conflict resolution should be an entire section entirely, but I can write up an example of one.